INZFS.MVC/Controllers/FundApplicationController.cs

```diff
@@ -341,6 +341,11 @@ public async Task<IActionResult> Save([Bind(Prefix = "submit.Publish")] string s
 
                 _session.Save(contentToSave);
 
+                if (currentPage.NextPageName != null)
```

Comment:
This is   likely   to break some   scenarios.

Reply:
The order of the next page check is important.

Reply:
Done, moved it last, so it won't override anything that's already there